packages/cli/test/Prompt.test.ts

```diff
@@ -0,0 +1,49 @@
+import * as Prompt from "@effect/cli/Prompt"
+import * as MockTerminal from "@effect/cli/test/services/MockTerminal"
+import type { Terminal } from "@effect/platform"
+import { Effect } from "effect"
+import * as Fiber from "effect/Fiber"
+import * as Layer from "effect/Layer"
+import { describe, expect, it } from "vitest"
+
+
+const MainLive = Effect.gen(function*(_) {
+  return Layer.mergeAll(
+    MockTerminal.layer,
+  )
+}).pipe(Layer.unwrapEffect)
```

Comment:
Given that `MockTerminal.layer` is already a `Layer` and we don't have any other layers to merge, there is no real reason to use `Effect.gen` here or `Layer.mergeAll`.

```suggestion
const MainLive = MockTerminal.layer
```